test/fuzz/fuzzer.ml

```diff
@@ -69,6 +74,20 @@ let compare (module I1 : Interprets.INTERPRET)
 
 let check (module I1 : Interprets.INTERPRET) (module I2 : Interprets.INTERPRET)
   m =
+  (* Save the generated module *)
+  if Param.save_modules then (
+    (* Create output directory if it doesn't exist *)
+    if not (Sys.file_exists Param.output_dir) then
+      Unix.mkdir Param.output_dir 0o755;
+
+    let filename = Printf.sprintf "%s/gen_do_module_%d.wat" 
```

Comment:
You can use `Fpath.(Param.output_dir / (Fmt.str "gen_do_module_%d !global_count))` instead to have proper directory separator on every platform.